cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -279,15 +279,15 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
 
     testQuery(
       title = "Create a node text index",
-      text = "A named text index on a single property for all nodes that have a particular label can be created with `CREATE TEXT INDEX index_name FOR (n:Label) ON (n.property)`. " +
+      text = "A named text index on string values in a single property for all nodes that have a particular label can be created with `CREATE TEXT INDEX index_name FOR (n:Label) ON (n.property)`. " +
```

Comment:
`on string values in a single property` reads weirdly to me. Not sure how I would formulate it though, perhaps someone in the docs team can give an suggestion.

  The same applies to the other three places as well.

Reply:
I didn't come up with any nice way of saying it but wanted it to be mentioned somehow

Reply:
How about moving the limitation to the _Note that text indexes..._ below the example?
That sentence would then read:
_Note that text indexes only recognize string values, do not support multiple properties, and that the index name must be unique._